galasa-extensions-parent/dev.galasa.ras.couchdb/src/test/java/dev/galasa/ras/couchdb/internal/mocks/MockEnvironment.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package dev.galasa.ras.couchdb.internal.mocks;
+
+import dev.galasa.framework.spi.Environment;
+
+import java.util.HashMap;
+import java.util.Map;
+public class MockEnvironment implements Environment{
+
+    private Map<String,String> envProps ;
+
+    public MockEnvironment() {
+    this.envProps = new HashMap<String,String>();
```

Comment:
Some formatting improvements here:
- A space is missing before the `{` in `implements Environment{`
- There's a space before the `;` in `envProps ;` which could be removed
- The statement in the constructor seems to be missing an indent